module_report/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Report Docx Module
+==================
+
+This module collect the data from current object and generate the pdf/docx report.
```

Comment:
This module adds a new reporting engine to the default ones and allow to generate report based on Docx template and Jinja2 scripting language.